operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -39,12 +39,12 @@ Product Managers are responsible for carrying out the strategic priorities of th
 * * What fundamental philosophies are predominant?
   * Pragmatic Marketing: “Your opinion, although interesting, is irrelevant.”
     * It’s all about the customer/user/buyer’s opinion/want/need
-* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\) 
+* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\)
```

Comment:
Could change the link to: https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq.